plugins/reach/lib/model/data/kModerationVendorTaskData.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @package plugins.reach
+ * @subpackage model
+ */
+class kModerationVendorTaskData extends kVendorTaskData
+{
+	public ?string $ruleIds = "";
```

Comment:
Change the type declarations to 'string', as '?string' is not supported by older PHP versions, and this might cause issues when creating clients.